Makefile

```diff
@@ -0,0 +1,33 @@
+export GO111MODULE := on
+export GOPROXY := go-proxy.oss.wandera.net
+export GONOSUMDB := github.com/wandera/*
+
+all: check test
+
+MAKEFLAGS += --no-print-directory
+
+prepare:
+	@echo "Downloading tools"
+	@cat tools.go | grep _ | cut -f2 -d " " | xargs -tI % sh -c "go install %"
+
+check: prepare
+	@echo "Running check"
+ifeq (, $(shell which golangci-lint))
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin v1.50.1
```

Comment:
minor: what about bumping to the newer version?

Reply:
sure, done. Will add github actions to automatically run linting and tests in next PR